### PR TITLE
Ignore listing by price

### PR DESCRIPTION
**Pull request by SeRi0uS007**
Ignore listing according to the final price.

Setup menu example
Setup menu example

"Sell all" example
"Sell all" example

At the moment, this pull request has a conflict with my another pull request #233. So, depending on what is accepted and how, I will fix my code.

### CODE REVIEW

File: code.user.js [hunk 1]
```diff
@@ -1296,17 +1298,23 @@
...
+                const itemName = task.item.name || task.item.description.name;
+                const padLeft = `${padLeftZero(`${totalNumberOfProcessedQueueItems}`, digits)} / ${totalNumberOfQueuedItems}`;
+
+                if (getSettingWithDefault(SETTING_PRICE_MIN_LIST_PRICE) * 100 >= market.getPriceIncludingFees(task.sellPrice)) {
+                    logDOM(`${padLeft} - ${itemName} is not listed due to ignoring price settings.`);
+                    $(`#${task.item.appid}_${task.item.contextid}_${itemId}`).css('background', COLOR_PRICE_NOT_CHECKED);
+                    next();
+
+                    return;
+                }
```
Comment (Nuklon) [inline code.user.js:1313]:
```suggestion
                totalNumberOfProcessedQueueItems++;
                
                const digits = getNumberOfDigits(totalNumberOfQueuedItems);
                const itemId = task.item.assetid || task.item.id;
                const itemName = task.item.name || task.item.description.name;
                const padLeft = `${padLeftZero(`${totalNumberOfProcessedQueueItems}`, digits)} / ${totalNumberOfQueuedItems}`;

                if (getSettingWithDefault(SETTING_PRICE_MIN_LIST_PRICE) * 100 >= market.getPriceIncludingFees(task.sellPrice)) {
                    logDOM(`${padLeft} - ${itemName} is not listed due to ignoring price settings.`);
                    $(`#${task.item.appid}_${task.item.contextid}_${itemId}`).css('background', COLOR_PRICE_NOT_CHECKED);
                    next();
                    return;
                }
                
```

**Comment by SeRi0uS007** [comment]
@Nuklon, I've updated and just tested my last commit to ensure it works.

Perhaps it also makes sense to create a separate const for stacked items?
```javascript
const itemNameWithAmount = task.item.amount == 1 ? itemName : `${task.item.amount}x ${itemName}`;
```

**Comment by Nuklon** [comment]
Thanks, looks good now.

**Comment by GGBot0** [comment]
Had this issue for a few weeks now, has this been resolved yet?

**Pull request merged**